src/server/acl/user.h

```diff
@@ -57,8 +57,11 @@ class User final {
     std::vector<UpdateKey> keys;
     bool reset_all_keys{false};
     bool allow_all_keys{false};
+
     // TODO allow reset all
     // bool reset_all{false};
+
+    std::optional<std::string> ns{};
```

Comment:
subjective nit (related to the previous comment) - can be just a string with empty indicating it's not set.

Reply:
SG. + @kostasrim to make sure this `UpdateRequest` class is always expected to contain all information (i.e. that there can't be only deltas in it)